docs_espressif/en/additionalfeatures/wsl.rst

```diff
@@ -0,0 +1,228 @@
+Using Docker Container
+===================================
+
+In this tutorial will show you how to develop your projects based on ``Visual Studio Code`` + ``ESP-IDF Extension`` + ``Remote - WSL`` to implement all features of this extension in WSL.
+
+Install the following tools before starting the project:
+
+1. Windows WSL (steps to install below).
+2. `Visual Studio Code <https://code.visualstudio.com>`_
+3. `usbipd-win <https://github.com/dorssel/usbipd-win/releases>`_
+
+Ubuntu on Windows
+---------------------
+
+If you don't have WSL installed run:
+
+.. code-block::
+
+  wsl --install
+
+Update the WSL kernel with
+
+.. code-block::
+
+  wsl --update
+
+Check the WSL available distributions list with the ``Powershell`` command prompt, as below
+
+.. code-block::
+
+  wsl -l -o
+
+.. image:: ../../../media/tutorials/using_docker_container/wsl-l-o.png
+
+so to install a ubuntu distribution in WSL on Windows, please type in the following command:
```

Comment:
On this page there are a lot of instances where we start sentences or steps with lower case letters, I think we should not do that. In some cases we might even need te re-phrase some of the steps in order to start with capital letters in my opinion